cmd/dlv/dlv_test.go

```diff
@@ -941,6 +941,175 @@ func TestTrace2(t *testing.T) {
 	assertNoError(cmd.Wait(), t, "cmd.Wait()")
 }
 
+func TestTraceDepth(t *testing.T) {
```

Comment:
I think it would be better to just have one of these end to end tests and then test the rest of the detailis in service/test/integration2_test.go by just calling ListFunctions directly.